portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Chris's Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Hobbies</h1>
+      <hr>
+      <section>
+        <nav>
+            <ul>
+                  <li><a class="one" href="index.html" style="color:#777;" >Home</a></li>
+                 <br>
+                 <li><a class="one" href="pictures.html" style="color:#777;">Pictures</a></li>
+                 <br>
+                 <li><a class="one" href="hobbies.html" style="color:#777;" >Hobbies</a></li>
+                 <br>                              
+                 <li><a class="one" href="resume.html" style="color:#777;" >Resume</a></li>
+                 <br>
```

Comment:
nit: Let's fix the indentation of the tags here a bit. 

FYI a "nit" is a prefix we use to indicate a "nitpick that we think would improve the code but isn't mandatory".